### PR TITLE
Update GitLab icon (2022)

### PR DESCRIPTION
**Pull request by tnir**
GitLab logo was updated on 2022-04-27 by @GitLab: https://about.gitlab.com/blog/2022/04/27/devops-is-at-the-center-of-gitlab/

### `.gitlab.path`

Original: https://about.gitlab.com/images/press/logo/svg/gitlab-logo-600.svg

Modified with `curl -o - -sL https://about.gitlab.com/images/press/logo/svg/gitlab-logo-600.svg | sed -E 's#</?(svg|g)[^>]*>##g` command to remove `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 380 380"></svg>` and `<g id="LOGO"></g>`.

We can test the changes as follows:
```bash
$ diff -u <(curl -o - -sL https://about.gitlab.com/images/press/logo/svg/gitlab-logo-600.svg | sed -E 's#</?(svg|g)[^>]*>##g';echo) <(jq -r .gitlab.path shared/ui/Stream/icons8-data.json)
```

### `.gitlab.viewBox`
The original SVG has huge margin so we need to crop it.

- cf. https://github.com/github/explore/pull/3059

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

### CODE REVIEW

**Comment by bcanzanella** [comment]
Thanks for the PR @tnir! We have to do a little housekeeping before we can accept PRs from the community, so please sit tight!

**Comment by tnir** [comment]
Is that different from kind of https://github.com/TeamCodeStream/codestream/blob/develop/CONTRIBUTING.md?

**Comment by bcanzanella** [comment]
> Is that different from kind of https://github.com/TeamCodeStream/codestream/blob/develop/CONTRIBUTING.md?

yes since we [CodeStream] are now a part of New Relic.

**Comment by tnir** [comment]
Yeah, I thought it WAS 6 months ago... so PMI was also finished 😀

File: shared/ui/Stream/icons8-data.json [hunk 1]
```diff
@@ -98,8 +98,8 @@
...
 		"width": 16,
 		"height": 16,
-		"viewBox": "0 0 500 500",
-		"path": "<path id=\"path50\" class=\"st0\" d=\"M249.9,476.8L249.9,476.8l90.7-279.1H159.2L249.9,476.8L249.9,476.8z\"/><path id=\"path66\" class=\"st0\" d=\"M32.1,197.7L32.1,197.7L4.5,282.5c-2.5,7.7,0.2,16.2,6.8,21l238.5,173.3L32.1,197.7L32.1,197.7z\"/><path id=\"path74\" class=\"st0\" d=\"M32.1,197.7h127.1L104.6,29.6c-2.8-8.6-15-8.6-17.9,0L32.1,197.7L32.1,197.7z\"/><path id=\"path82\" class=\"st0\" d=\"M467.6,197.7L467.6,197.7l27.6,84.8c2.5,7.7-0.2,16.2-6.8,21L249.9,476.8L467.6,197.7L467.6,197.7z\"/><path id=\"path86\" class=\"st0\" d=\"M467.6,197.7H340.5l54.6-168.1c2.8-8.6,15-8.6,17.9,0L467.6,197.7L467.6,197.7z\"/>"
+		"viewBox": "93 93 194 194",
+		"path": "<defs><style>.cls-1{fill:#171321;}</style></defs><path class=\"cls-1\" d=\"M282.83,170.73l-.27-.69-26.14-68.22a6.81,6.81,0,0,0-2.69-3.24,7,7,0,0,0-8,.43,7,7,0,0,0-2.32,3.52l-17.65,54H154.29l-17.65-54A6.86,6.86,0,0,0,134.32,99a7,7,0,0,0-8-.43,6.87,6.87,0,0,0-2.69,3.24L97.44,170l-.26.69a48.54,48.54,0,0,0,16.1,56.1l.09.07.24.17,39.82,29.82,19.7,14.91,12,9.06a8.07,8.07,0,0,0,9.76,0l12-9.06,19.7-14.91,40.06-30,.1-.08A48.56,48.56,0,0,0,282.83,170.73Z\"/>"
```
Comment (bcanzanella) [inline shared/ui/Stream/icons8-data.json:102]: This looks good, but we don't want to have hard-coded color styles and we don't need the inline class. Can you make this change? (see suggestion)

```suggestion
		"path": "<path d=\"M282.83,170.73l-.27-.69-26.14-68.22a6.81,6.81,0,0,0-2.69-3.24,7,7,0,0,0-8,.43,7,7,0,0,0-2.32,3.52l-17.65,54H154.29l-17.65-54A6.86,6.86,0,0,0,134.32,99a7,7,0,0,0-8-.43,6.87,6.87,0,0,0-2.69,3.24L97.44,170l-.26.69a48.54,48.54,0,0,0,16.1,56.1l.09.07.24.17,39.82,29.82,19.7,14.91,12,9.06a8.07,8.07,0,0,0,9.76,0l12-9.06,19.7-14.91,40.06-30,.1-.08A48.56,48.56,0,0,0,282.83,170.73Z\"/>"
```

**Comment by bcanzanella** [comment]
@tnir we're now ready for your PR. You'll have to sign the CLA and I've added a comment for you to address.

**Comment by tnir** [comment]
Thanks. Confirmed as resolved in my production (my dev env):

codestream-with-new-gitlab-logo

It looks faster than [GitLab VS Code Extension (or recently renamed to GitLab Workflow?)](https://marketplace.visualstudio.com/items?itemName=GitLab.gitlab-workflow)

**Pull request merged**